Automattic/woocommerce-payments
# Render a notice when the available balance is below the minimum deposit threshold

Fixes #6774

### Changes proposed in this Pull Request

This PR renders a notice on the **Payments → Overview → Deposits** card when the available balance for the selected currency is above zero yet below the minimum deposit amount.

Minimum deposit amounts are defined per currency and made available to the UI via the `wcpaySettings` global JS object.


**TODO:**
- [x] Testing instructions
- [x] Check on mobile
- [x] Check against server PR https://github.com/Automattic/woocommerce-payments-server/pull/4308

### Testing instructions





* Test the following scenarios and visit the Payments → Overview screen to ensure the notice appears when expected:

**Available balance below minimum deposit threshold**
* A notice should appear with the correct minimum balance shown for the selected currency.
* The deposit schedule message should not appear
<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/6c36f9a5-52d3-4803-a2a9-1e8dc43a75e2">

**Available balance above minimum deposit threshold**
* The minimum deposit threshold notice should **not** appear.
* The deposit schedule message should appear.
<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/e8abaced-f8af-465b-b8f0-2c41bce72f40">



-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : **Not provided due to low risk and lengthy setup**
- [x] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable. **Added to what's new post**
